include/mbedtls/config_psa.h

```diff
@@ -1015,6 +1015,58 @@ extern "C" {
 #define PSA_WANT_ALG_SOME_PAKE 1
 #endif
 
+/* Backward compatibility */
+#if defined(PSA_WANT_KEY_TYPE_ECC_KEY_PAIR_USE) && \
+    defined(PSA_WANT_KEY_TYPE_ECC_KEY_PAIR_IMPORT) && \
+    defined(PSA_WANT_KEY_TYPE_ECC_KEY_PAIR_EXPORT) && \
+    defined(PSA_WANT_KEY_TYPE_ECC_KEY_PAIR_GENERATE)
+//defined(PSA_WANT_KEY_TYPE_ECC_KEY_PAIR_DERIVE) /* Not supported yet */
```

Comment:
I think it is supported for ECC - just not through drivers yet.

Reply:
Yes, sorry, I made some unnecessary comments at the beginning because I "mixed" the derivation feature for RSA with the one for ECC. Unfortunately I didn't fix them all apparently